server/api/actions/users.js

```diff
@@ -7,16 +7,16 @@ const getUsers = async () => {
 
 const createUser = async ({name, identifier}) => {
   const { rows: data } = await db.execute('server/sql/users/put.sql', {name, identifier});
-  return data;
+  return data.length ? data[0] : null;
 };
```

Comment:
we can do even more destructuring here:

```javascript
const createUser = async ({name, identifier}) => {
  const { rows: [data] } = await db.execute('server/sql/users/put.sql', {name, identifier});
  return data;
};
```

if there is no `data` object in the first indexed array element, then it will just return `null`

Reply:
Wow.